examples/get-content-from-multiple-elements/README.md

```diff
@@ -0,0 +1,23 @@
+# Get Content From Multiple Elements
+
+**Test Code**: [index.js](index.js)
+
+**Tested Page**: [index.html](index.html)
+
+This example shows how to extract text content from all items in an `<ul>` list.
```

Comment:
I guess we extract content from each item, one by one. Also, `an 'ul' list` doesn't work it for me (an unordered list list). I'd say:

```This example shows how to extract text content from every item in an `<ul>` element.```

**OR**

```This example shows how to extract text content from every item in an unordered list (`<ul>`).```